examples/recorded_moves_example.py

```diff
@@ -2,14 +2,20 @@
 
 Run :
 
-python3 recorded_moves_example.py -l [dance, emotions]
+python3 recorded_moves_with_url.py -l [dance, emotions]
```

Comment:
This should ne re renamed to `recorded_moves_examples.py`, `recorded_moves_with_url.py` doesn't exist